index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
Sure I can, but I actually tried first with yellow, and the contrast with the white background didn't fit too well. Anyway I will later take a screenshot of both and paste it here, so we can check it.